src/server.ts

```diff
@@ -0,0 +1,14 @@
+import { typeDefs } from './schema';
+import { resolvers } from './resolver';
+import { ApolloServer } from '@apollo/server';
+import { startStandaloneServer } from '@apollo/server/standalone';
+
+export async function setupServer() {
+  const server = new ApolloServer({ typeDefs, resolvers });
+
+  startStandaloneServer(server, {
+    listen: { port: 4000 },
+  }).then(({ url }) => {
+    console.log(`🚀  Server ready at: ${url}`);
+  });
```

Comment:
Aqui tem um pequeno probleminha de programação assíncrona. Como vc tá usando o then, quem implementar essa função vai continuar rodando depois do comando `startStandaloneServer` normalmente, sem esperar ele retornar. Pra que isso não aconteça, o ideal é que vc use await aqui tbm:

```suggestion
  const { url } = await startStandaloneServer(server, { listen: { port: 4000 } });
  console.log(`🚀  Server ready at: ${url}`);
```